wagtail/wagtailredirects/tests.py

```diff
@@ -16,26 +16,62 @@ def test_path_normalisation(self):
         path = normalise_path('/Hello/world.html;fizz=three;buzz=five?foo=Bar&Baz=quux2')
 
         # Test against equivalant paths
-        self.assertEqual(path, normalise_path('/Hello/world.html;fizz=three;buzz=five?foo=Bar&Baz=quux2'))  # The exact same URL
-        self.assertEqual(path, normalise_path('http://mywebsite.com:8000/Hello/world.html;fizz=three;buzz=five?foo=Bar&Baz=quux2'))  # Scheme, hostname and port ignored
-        self.assertEqual(path, normalise_path('Hello/world.html;fizz=three;buzz=five?foo=Bar&Baz=quux2'))  # Leading slash can be omitted
-        self.assertEqual(path, normalise_path('Hello/world.html/;fizz=three;buzz=five?foo=Bar&Baz=quux2'))  # Trailing slashes are ignored
-        self.assertEqual(path, normalise_path('/Hello/world.html;fizz=three;buzz=five?foo=Bar&Baz=quux2#cool'))  # Fragments are ignored
-        self.assertEqual(path, normalise_path('/Hello/world.html;fizz=three;buzz=five?Baz=quux2&foo=Bar'))  # Order of query string parameters is ignored
-        self.assertEqual(path, normalise_path('/Hello/world.html;buzz=five;fizz=three?foo=Bar&Baz=quux2'))  # Order of parameters is ignored
-        self.assertEqual(path, normalise_path('  /Hello/world.html;fizz=three;buzz=five?foo=Bar&Baz=quux2'))  # Leading whitespace
-        self.assertEqual(path, normalise_path('/Hello/world.html;fizz=three;buzz=five?foo=Bar&Baz=quux2  '))  # Trailing whitespace
+        self.assertEqual(path, normalise_path(  # The exact same URL
```

Comment:
I'm happy with this but perhaps this code could be made even clearer by putting the URLs to test against in a list and do the assertion in a for loop.

No need to do that in this PR though